apps/main/src/dex/store/createUserBalancesSlice.ts

```diff
@@ -57,17 +59,29 @@ const createUserBalancesSlice = (
       const filteredBadTokens = tokensAddresses.filter((address) => !excludeTokensBalancesMapper[address])
 
       sliceState.setStateByKey('loading', true)
-      const userBalancesMapper = await curvejsApi.wallet.fetchUserBalances(curve, filteredBadTokens)
+
+      // This gets multicall batched by Wagmi and Viem internally
```

Comment:
imo this should be a function in token-balance.query.ts

Reply:
Decided not too, because the usage of `Promise.allSettled` and porting over the results to the `userBalancesMapper` is highly situation to the store. It's not something I want to export for other to re-use.